sound/attributions.txt

```diff
@@ -94,6 +94,8 @@ https://www.zapsplat.com/sound-effect-category/sleigh-bells/
 tada_fanfare.ogg is adapted from plasterbrain's "Tada Fanfare A", which is public domain (CC 0):
 https://freesound.org/people/plasterbrain/sounds/397355/
 
+mountedgun.ogg and mountedgunend.ogg are most likely from a stock sound of a magnum firing ,"44 MAGNUM A", from sound ideas 6000. However, its heavily truncated, so could be any number of gunshots.
```

Comment:
it would be nice to get an actual link to the original source :(

Reply:
I'll keep digging around, but I cant promise anything. The original ogg is just called gun_08 with zero metadata so I dont have high hopes.

Reply:
i don't think we can accept a completely unsourced file, unless you're saying it's already in the codebase.

Reply:
Understandable, ill see if I can find an alternative if unable to find an exact source. I have some examples it COULD be, but there are a lot of gun sounds out there that are nearly indistinguishable.

Reply:
Ok new sounds in that wont get us sued (combinations of cannon and revolver)